jlm/util/Makefile.sub

```diff
@@ -1,6 +1,7 @@
 libutil_SOURCES = \
```

Comment:
You forgot to add the header file to the Makefile.

Reply:
Yes, but this `Makefile.sub` has no headers listed. Is this a mistake? Do you want me to add them all?

Reply:
See PR #404.

Reply:
Once merged, just add the header file to it, please.